cluster/common.sh

```diff
@@ -603,6 +603,7 @@ KUBELET_TOKEN: $(yaml-quote ${KUBELET_TOKEN:-})
 KUBE_PROXY_TOKEN: $(yaml-quote ${KUBE_PROXY_TOKEN:-})
 ADMISSION_CONTROL: $(yaml-quote ${ADMISSION_CONTROL:-})
 MASTER_IP_RANGE: $(yaml-quote ${MASTER_IP_RANGE})
+MASTER_RESERVED_IP: $(yaml-quote ${MASTER_RESERVED_IP})
```

Comment:
> We will be reserving IP only on GCE, but here you're adding this to all deployments. 

I think we should just re-use the existing variable (`KUBERNETES_MASTER_NAME`) and set it to the appropriate value (string, ip address, etc) based on the deployment. For GCE, we can just start passing in the reserved ip address.